hazelcast-client/src/test/java/com/hazelcast/client/cache/impl/nearcache/invalidation/InvalidationMemberAddRemoveTest.java

```diff
@@ -70,6 +75,14 @@
     private static final int KEY_COUNT = 1000;
     private static final int RECONCILIATION_INTERVAL_SECONDS = 30;
 
+    @Parameter
+    public LocalUpdatePolicy localUpdatePolicy;
+
+    @Parameters
+    public static LocalUpdatePolicy[] localUpdatePolicies() {
```

Comment:
A name can be added to parameter like `localUpdatePolicy` 
Also testing CACHE_ON_UPDATE can be sufficient since CACHE is the deprecated name of the same functionality.

Final version after changes can be something like this: 
```
    @Parameters(name = "localUpdatePolicy:{0}")
    public static Collection<Object[]> parameters() {
        return asList(new Object[][]{
                {INVALIDATE},
                {CACHE_ON_UPDATE}
        });
    }
```